src/Tokenizer/CT.php

```diff
@@ -74,6 +76,8 @@ public static function getName(int $value): string
 
         $tokens = self::getMapById();
 
+        \assert(isset($tokens[$value]));
```

Comment:
@kubawerlos you were suggesting me better approaches for `asserts` few times. any idea here?